.github/workflows/merged.yml

```diff
@@ -85,7 +85,7 @@ jobs:
           echo -n "${{env.AWS_SECRET_DISTRIBUTION_P12_ENCODED}}" | base64 --decode -o $CERTIFICATE_PATH
           echo -n "${{env.AWS_SECRET_AUTH_KEY_P8_ENCODED}}" | base64 --decode -o $APIKEY_PATH
           
-          bundle exec fastlane prerelease configuration:"Debug" \
+          bundle exec fastlane prerelease configuration:"Staging" \
```

Comment:
Just want to check my understanding here @DylanRJ - does this mean that on merge we will push the staging version of the app (ie. the one that calls real auth) to Test Flight?

Reply:
Yes, currently.
We can establish deploying to both staging and build apps now that we have both apps established, will need just need the back end to reflect real auth as we just discussed but then the journeys should both be fine.
That change is obviously separate to this so I will add that deploy to the Build app as additional commit to the PR?
Happy to do it as a separate PR if you'd like